app/views/layouts/internal.html.erb

```diff
@@ -40,7 +40,7 @@
 
   <div class="container mt-3">
     <% flash.each do |type, message| %>
-      <div class="alert alert-<%= type == "notice" ? "success" : "danger" %>">
+      <div class="alert alert-<%= type == "notice" || type == "success" ? "success" : "danger" %>">
```

Comment:
We may need to revisit this in another PR, there are definitely some oddities here!

This works for now :+1: